legal-api/src/legal_api/reports/report.py

```diff
@@ -684,9 +684,9 @@ def _format_alteration_data(self, filing):
             new_legal_type = filing.get('alteration').get('business').get('legalType')
             prev_legal_name = filing.get('business').get('legalName')
             identifier = filing.get('business').get('identifier')
-            name_request_json = filing.get('alteration').get('nameRequest')
-            if name_request_json:
-                to_legal_name = self._create_numbered_legal_name(new_legal_type, identifier)
+            name_request_json = filing.get('alteration').get('nameRequest', {})
+            if name_request_json and not (to_legal_name := name_request_json.get('legalName')):
```

Comment:
Won't `name_request_json` always be True, since you assigned a fallback empty object above?

Reply:
Nope. In python it need value to be True
![image](https://github.com/user-attachments/assets/e1b585d7-5db6-49b3-96b9-41c590dbcf0d)